Car-slider.html

```diff
@@ -0,0 +1,238 @@
+        color: #484848;
+        font-weight: 400;
+      }
+
+      .car__price {
+        font-size: 16px;
+        margin-top: 8px;
+        font-weight: 700;
+        color: #484848;
+      }
+
+      .car__avg__price {
+        font-size: 13px;
+        margin-top: 8px;
+        color: #6f6f6f;
+      }
+
+      .car__city__price {
+        margin-top: 8px;
+      }
+
+      .citywise__price {
+        color: #0288d1;
+        text-decoration: none;
+      }
+
+      .onwards__text {
+        font-size: 13px;
+        font-weight: 400;
+        color: #aaa;
+        padding-top: 2px;
+        padding-left: 3px;
+      }
+
+      .previous__card__slider,
+      .next__card__slider {
+        height: 32px;
+        width: 32px;
+        background-color: #d7d7d7;
+        position: absolute;
+        z-index: 1;
+        top: 50%;
+        border-radius: 50%;
+        display: flex;
+        justify-content: center;
+        align-items: center;
+      }
+
+      .next__card__slider {
+        right: 0;
+      }
+
+      .previous__arrow {
+        transform: rotate(180deg);
+      }
+
+      .previous__arrow,
+      .next__arrow {
+        width: 16px;
+        height: 16px;
+        color: #484848;
+      }
+
+      .car__slider__wrapper {
+        display: flex;
+        overflow-x: scroll;
+      }
+
+      ::-webkit-scrollbar {
+        width: 4px;
+        height: 1px;
+      }
+      ::-webkit-scrollbar-thumb {
+        background-color: #c5c5c5;
+        border-radius: 12px;
+        border: 1px solid #f6f6f6;
+      }
+      ::-webkit-scrollbar-track {
+        background: #f6f6f6;
+      }
+    </style>
+  </head>
+  <body>
+    <div class="card__container">
+      <a class="previous__card__slider" href="#box1">
+        <svg
+          class="previous__arrow"
+          viewBox="0 0 16 16"
+          fill="currentcolor"
+          tabindex="-1"
+          focusable="false"
+          aria-hidden="true"
+          role="img"
+        >
+          <path
+            d="M4.59 15.8a1 1 0 01-.69-.29 1 1 0 010-1.37L10 8 3.9 1.86a1 1 0 010-1.37 1 1 0 011.38 0l6.82 6.82a1 1 0 010 1.38l-6.82 6.82a1 1 0 01-.69.29z"
+          ></path>
+        </svg>
+      </a>
+
+      <div class="car__slider__wrapper">
+        <div class="car__info__wrapper" id="box1">
+          <div class="card__image__content">
+            <img
+              src="https://imgd.aeplcdn.com/310x174/n/cw/ec/40087/thar-exterior-right-front-three-quarter-11.jpeg?q=75"
+              alt="Mahindra Thar"
+            />
+            <div class="car__information">
+              <p class="car__name">Mahindra Thar</p>
+              <p class="car__price">
+                ₹ 9.9 Lakh <span class="onwards__text">onwards</span>
+              </p>
+              <p class="car__avg__price">Avg Ex showroom price</p>
+              <p class="car__city__price">
+                <a href="#" class="citywise__price">Show price in my city</a>
+              </p>
```

Comment:
```suggestion
                <a href="#" class="card__city-price-link">Show price in my city</a>
```
not required wrapping of p tag